commit 5e669fe7e986c1c7284ea5e8bf3662e05742d7b0 (refs/changes/19/4919/1)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2018-10-02T21:17:18-07:00 (1 year ago)
    
    TRITON-822 DAPI should have a filter which can reliably fail a provision

diff --git a/package.json b/package.json
index 2f079ba..0fb72bf 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "SmartDataCenter Compute Node API",
-  "version": "1.12.2",
+  "version": "1.13.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -9,7 +9,7 @@
     "assert-plus": "1.0.0",
     "async": "1.5.2",
     "bunyan": "1.8.5",
-    "dapi": "git+https://github.com/joyent/sdc-designation.git#45a8d7c367300d3003838a4e0241915cf89fe3f9",
+    "dapi": "git+https://github.com/joyent/sdc-designation.git#grr-TRITON-822",
     "deep-equal": "1.0.1",
     "dox": "0.9.0",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
diff --git a/sapi_manifests/cnapi/template b/sapi_manifests/cnapi/template
index 48d749b..d365a15 100644
--- a/sapi_manifests/cnapi/template
+++ b/sapi_manifests/cnapi/template
@@ -81,6 +81,7 @@
 			"pipe", "hard-filter-setup",
 			        "hard-filter-running",
 			        "hard-filter-invalid-servers",
+			        "hard-filter-force-failure",
 			        "hard-filter-volumes-from",
 			        "hard-filter-reserved",
 			        "hard-filter-min-free-disk",
